runtime/js/99_main.js

```diff
@@ -349,8 +364,11 @@ delete Object.prototype.__proto__;
       runtimeOptions,
       internalName ?? name,
     );
-    const { unstableFlag, pid, noColor, args } = runtimeOptions;
+    const { unstableFlag, pid, noColor, args, location: locationHref } =
+      runtimeOptions;
 
+    location.setLocationHref(locationHref);
```

Comment:
On line 305 this is wrapped in a conditional. Why not here?

Reply:
Workers are unconditionally given a location.

Reply:
What location? So we align with browsers here?

Reply:
Yes, the location in a worker is the URL of its entrypoint module.